driver/web/rest/adminapis.go

```diff
@@ -214,3 +215,39 @@ func (h AdminApisHandler) ClearTestSentNudges(l *logs.Log, claims *tokenauth.Cla
 	}
 	return l.HttpResponseSuccess()
 }
+
+//FindNudgesProcesses gets all the nudges-process
+func (h AdminApisHandler) FindNudgesProcesses(l *logs.Log, claims *tokenauth.Claims, w http.ResponseWriter, r *http.Request) logs.HttpResponse {
+	var err error
+
+	//limit and offset
+	limit := 5
+	limitArg := r.URL.Query().Get("limit")
+	if limitArg <= "5" {
```

Comment:
If the client has passed a limit then use it but do not check if < 5. Use 5 only when the client does not pass limit.